source/polling-srv-records-for-mongos-discovery/polling-srv-records-for-mongos-discovery.rst

```diff
@@ -9,8 +9,8 @@ Polling SRV Records for mongos Discovery
 :Author: Derick Rethans
 :Status: Accepted
 :Type: Standards
-:Last Modified: 2018-11-29
-:Version: 1.0
+:Last Modified: 2021-09-03
+:Version: 1.1
```

Comment:
Maybe this should just be a patch bump; LMK what you think.

Reply:
I think a minor version bump makes sense, since this is very much a new feature for drivers.